Baby Monitor/Source Files/Localizations/en.lproj/Localizable.strings

```diff
@@ -57,14 +59,18 @@
 "onboarding.pairing.connecting.description" = "Searching for\nsecond device";
 "onboarding.pairing.available-devices.title" = "Available devices";
 "onboarding.pairing.refresh.button" = "Refresh the list";
+"onboarding.pairing.compare-code" = "Compare the security code and accept on the Baby device";
 "onboarding.pairing.done.description" = "Done!";
 "onboarding.pairing.error-description" = "Ups! Something\nwent wrong :(";
 "onboarding.pairing.error-second-description" = "Make sure that both devices are\nconnected to the same WiFi network.";
+"onboarding.pairing.connection-error-second-description" = "The baby device declined the connection.";
```

Comment:
There is inconsistency between capitalization in `Compare the security code and accept on the Baby device` and `The baby device declined the connection.` - is it on purpose?

Reply:
Good catch!